apps/web/next.config.mjs

```diff
@@ -1,4 +1,42 @@
+import path from 'path'
+import { fileURLToPath } from 'url'
+
+const currentFileURL = import.meta.url
+const __filename = fileURLToPath(currentFileURL)
+const __dirname = path.dirname(__filename)
+
+const PACKAGES = ['env', 'core', 'jobs']
+const PACKAGE_NAMES = PACKAGES.map((pkg) => `@latitude-data/${pkg}`)
+
+function buildAlias({ localPackages }) {
+  return localPackages.reduce((aliases, pkg) => {
+    aliases[`$${pkg}`] = path.resolve(__dirname, `../../packages/${pkg}/src`)
+    return aliases
+  }, {})
+}
```

Comment:
If we go for the Rollup build by package in production we check NODE_ENV and avoid setting these alias. That's all it needs I think

Reply:
yes please 🙏🏼